packages/requester-fetch/src/__tests__/unit/fetch-requester.test.ts

```diff
@@ -0,0 +1,271 @@
+import { MethodEnum, Request } from '@algolia/requester-common';
+import nock from 'nock';
+// @ts-ignore
+import nodeFetch from 'node-fetch';
+// @ts-ignore
+import { Readable } from 'readable-stream';
```

Comment:
It seems that `@types/readable-stream` does not work well because it supports readable-stream v2, not readable-stream v3. Also, `@types/node-fetch` returns a type incompatible with the fetch API, so it has been replaced with `cross-fetch` (a wrapper for `node-fetch`).